src/sql/inspect.py

```diff
@@ -162,8 +228,12 @@ def __init__(self, table_name, schema=None) -> None:
                     GROUP BY top ORDER BY frequency Desc""",
                 ).fetchall()
 
-                table_stats[column]["freq"] = result_col_freq_values[0][1]
-                table_stats[column]["top"] = result_col_freq_values[0][0]
+                if is_numeric is False:
+                    table_stats[column]["freq"] = result_col_freq_values[0][1]
+                    table_stats[column]["top"] = result_col_freq_values[0][0]
+                else:
+                    table_stats[column]["freq"] = math.nan
```

Comment:
If column is numeric type we are setting frequency and top = NaN. Do we know why `pandas` does this?

Reply:
So frequency and top are usually stats reserved for categorical data. 
If i have a column with 5 unique values `car, bike plane cycle, bus`.
Top indicates the value that has appeared the most in the data 
freq indicated how many times the `top` value appears in the data.

These stats don't translate to numerical data because in a numerical column, two data points usually won't be the same. Most of the data points will be unique, so `Top` and `Freq` are set to Nan.